pkgs/os-specific/linux/firejail/whitelist-nix-profile.patch

```diff
@@ -1,9 +0,0 @@
---- a/etc/inc/whitelist-common.inc.org	2022-05-06 13:57:17.294206339 +0200
-+++ b/etc/inc/whitelist-common.inc	2022-05-06 13:58:00.108655548 +0200
-@@ -83,3 +83,6 @@
- whitelist ${HOME}/.kde4/share/config/oxygenrc
- whitelist ${HOME}/.kde4/share/icons
- whitelist ${HOME}/.local/share/qt5ct
-+
-+# NixOS specific to resolve binary paths
-+whitelist ${HOME}/.nix-profile
```

Comment:
This was added through https://github.com/netblue30/firejail/commit/bd0c18b1bf74bb5b1db2670734c2de8009c5c3d6
which is NOT contained in the version nixpkgs currently uses
https://github.com/NixOS/nixpkgs/blob/0db6f20cfeb470c1f778442217c39d8202d591bd/pkgs/os-specific/linux/firejail/default.nix#L14
```
$ git branch --contains bd0c18b1bf74bb5b1db2670734c2de8009c5c3d6
master
```